server/portal/libs/agave/operations.py

```diff
@@ -225,14 +225,20 @@ def move(client, src_system, src_path, dest_system, dest_path, file_name=None):
         return {'system': src_system, 'path': src_path_full, 'name': file_name}
 
     try:
-        client.files.list(systemId=dest_system,
-                          filePath="{}/{}".format(dest_path, file_name))
-
-        # Destination path exists, must make it unique.
-        _ext = os.path.splitext(file_name)[1].lower()
-        _name = os.path.splitext(file_name)[0]
-        now = datetime.datetime.utcnow().strftime('%Y-%m-%d %H-%M-%S')
-        file_name = '{}_{}{}'.format(_name, now, _ext)
+        # list the directory and check if file_name exists
+        file_listing = client.files.list(systemId=dest_system, filePath=dest_path)
+
+        if len([x['name'] for x in file_listing if x['name'] == file_name]) > 0:
+            inc = 1
+            _ext = os.path.splitext(file_name)[1].lower()
```

Comment:
should the lower should be removed; i guess i'm not following why we would want to change it.

Reply:
To be honest I'm not sure either. I thought looked unusual, so I assumed it must have been there to catch some extraordinary cases where the file extension would be in all caps or something.

Reply:
saw this is resolved in new utility function 👍